sale_weekday_delivery/readme/ROADMAP.rst

```diff
@@ -0,0 +1 @@
+* Fix computation of _compute_qty_at_date on sale.order.line
```

Comment:
TODO: 

 - Use onchange to define commitment_date (from partner_shipping_id)
 - Redefine commitment_date on confirmation
 - Add a computed field to state if commitment_date is set manually or through Fixed weekdays preference.

Reply:
Instead of messing with `commitment_date`, we can use following hook to ease date computation per line: https://github.com/odoo/odoo/pull/48322